tensorflow/c/experimental/filesystem/filesystem_interface.h

```diff
@@ -893,7 +899,9 @@ typedef struct TF_FilesystemOps {
   ///
   /// DEFAULT IMPLEMENTATION: return `TF_NOT_FOUND`.
   void (*set_filesystem_configuration_option)(
-      const TF_Filesystem* filesystem, const TF_Filesystem_Option* option,
+      const TF_Filesystem* filesystem,
+      const char* key,
```

Comment:
The key is specified in `option` (`option->name`), so we don't need a new argument here.

Reply:
@mihaimaruseac Updated.